tests/integration/bond_test.py

```diff
@@ -69,3 +69,23 @@ def test_remove_bond_with_minimum_desired_state():
     netapplier.apply(remove_bond_state)
     state = statelib.show_only((state[INTERFACES][0]['name'],))
     assert not state[INTERFACES]
+
+
+def test_add_bond_without_slaves():
+    desired_bond_state = {
+            INTERFACES: [
+                {
+                    'name': 'bond99',
+                    'type': 'bond',
+                    'state': 'up',
+                    'link-aggregation': {
+                        'mode': 'balance-rr',
+                        'slaves': []
+                    },
+                }
+
+            ]
+        }
+
+    netapplier.apply(copy.deepcopy(desired_bond_state))
```

Comment:
Please separate between the tested action and the assert with a new-line for readability.